gulp-tasks/build.js

```diff
@@ -43,6 +45,28 @@ const buildPackage = (projectPath) => {
     });
 };
 
+gulp.task('build:shared', () => {
+  return Promise.all([
+    buildJSBundle({
+      rollupConfig: {
+        entry: path.join(__dirname, '..', 'lib', 'log-helper.js'),
+        format: 'umd',
+        moduleName: 'goog.logHelper',
+        plugins: [
```

Comment:
Are these plugins needed? I don't see any external `require()`s in `log-helper.js`.